documentation/docs/framework/tags.md

```diff
@@ -167,6 +167,10 @@ class MyTestClass : FunSpec()
 Note that when you use these annotations you pass the tag string name, not the tag itself. This is due to Kotlin annotations only allow "primitive" arguments
 :::
 
+### Inheriting tags
+
+By default, the `@Tags` annotation will only be considered on the immediate Spec which it was applied to. However, a Spec can also inherit tags from superclasses and superinterfaces. To enable this, toggle `tagInheritance = true` in your [project config](./project-config.html)
```

Comment:
Is this the documentation that goes to production?
If so, we should release that only when the next version is ready

Reply:
It will be in the "next" version until we perform the release and cut the docs, at which point it'll show in the stable docs. 

Next version shows like this:

![image](https://user-images.githubusercontent.com/316857/189229593-63e4e4a2-d05b-4887-a4ea-de87009508d6.jpeg)